chromium/browser_type_test.go

```diff
@@ -111,17 +109,6 @@ func TestBrowserTypePrepareFlags(t *testing.T) {
 			changeK6Opts:  &k6lib.Options{},
 			expChangedVal: nil,
 		},
-		{
-			flag:       "enable-use-zoom-for-dsf",
```

Comment:
Oops, we all missed this 😅

Reply:
Oh tricky case, I was wondering why it was not failing in CI or in my local machine, then a saw the skip when `runtime.GOOS != "darwin"`.

Reply:
That'll teach me for not running tests 😅